CHANGELOG.md

```diff
@@ -1,5 +1,133 @@
 # Changelog
 
+## v0.66.0
+
+### Highlights
+- Support for iOS 15 with Xcode 13 and Android 12 (TODO)
+- Hermes 0.9.0
+  - This release is primarily about closing gap between Hermes cut and this React Native release. Among ~400 commits, contains memory and size wins, bugfixes and other progress behind the scenes. See [issue for more details](https://github.com/facebook/hermes/issues/586).
```

Comment:
> This release is primarily about closing gap between Hermes cut and this React Native release.

Do we need this sentence?

Reply:
Added it to set context of the release -- that it's not a release motivated by major changes.

Reply:
Same comment I've put in the blogpost draft: I think it might be better to clarify what "this release" means - by reading Xuan's post it's clear that it means "This Hermes release" but within the context of the blogpost it seems to imply "this RN release". So maybe worth changing to "This Hermes release"? :)
```suggestion
  - This Hermes release is primarily about closing gap between Hermes cut and this React Native release. Among ~400 commits, contains memory and size wins, bugfixes and other progress behind the scenes. See [issue for more details](https://github.com/facebook/hermes/issues/586).
```